lib/src/list_multimap/list_multimap_builder.dart

```diff
@@ -68,6 +58,21 @@ class ListMultimapBuilder<K, V> {
     updates(this);
   }
 
+  /// Replaces all elements with elements from a [Map], [ListMultimap] or
+  /// [BuiltListMultimap].
+  void replace(multimap) {
```

Comment:
Setters without getters are a bit weird, and a getter wouldn't work here because it would have to have type dynamic. (Having setter and getter with same name but different types triggers an analyzer warning).